flygym/examples/olfaction/plume_tracking_arena.py

```diff
@@ -39,12 +41,36 @@ class OdorPlumeArena(BaseArena):
     def __init__(
         self,
         plume_data_path: Path,
+        main_camera_name: str,
         dimension_scale_factor: float = 0.5,
         plume_simulation_fps: float = 200,
         intensity_scale_factor: float = 1.0,
         friction: tuple[float, float, float] = (1, 0.005, 0.0001),
         num_sensors: int = 4,
     ):
+        """
+        Parameters
+        ----------
+        plume_data_path : Path
+            Path to the HDF5 file containing the plume simulation data.
+        main_camera_name : str
+            Name of the main camera used to render the plume.
+        dimension_scale_factor : float, optional
+            Scaling factor for the plume simulation grid. Each cell in the
+            plume grid is this many millimeters in the simulation. By
+            default 0.5.
+        plume_simulation_fps : float, optional
+            Frame rate of the plume simulation. Each frame in the plume
+            dataset is ``1 / plume_simulation_fps`` seconds in the physics
+            simulation. By default 200.
+        intensity_scale_factor : float, optional
+            Scaling factor for the intensity of the odor. By default 1.0.
+        friction : tuple[float, float, float], optional
+            Friction parameters for the floor geom. By default (1, 0.005,
+            0.0001).
+        num_sensors : int, optional
+            Number of olfactory sensors on the fly. By default 4.
+        """
```

Comment:
For me this docstring isn't needed because you already have it for the class. Is there some case where this doesn't work and you needed to duplicate the docstring here?

Reply:
removed in b49feef